PF4 - typedefs out of sync

Noted this a bit ago... will clean up soon if I get a chance...

```
ERROR in /Users/priley/GitHub/web-ui/frontend/node_modules/@patternfly/react-core/dist/js/components/EmptyState/EmptyState.d.ts(4,34):
TS1254: A 'const' initializer in an ambient context must be a string or numeric literal.
ERROR in /Users/priley/GitHub/web-ui/frontend/node_modules/@patternfly/react-core/dist/js/components/Select/CheckboxSelectGroup.d.ts(2,22):
TS2307: Cannot find module '../../typeUtils'.
ERROR in /Users/priley/GitHub/web-ui/frontend/node_modules/@patternfly/react-core/dist/js/components/Select/CheckboxSelectOption.d.ts(2,22):
TS2307: Cannot find module '../../typeUtils'.
ERROR in /Users/priley/GitHub/web-ui/frontend/node_modules/@patternfly/react-core/dist/js/components/Select/Select.d.ts(8,18):
TS2430: Interface 'SelectProps' incorrectly extends interface 'HTMLProps<HTMLOptionElement>'.
  Types of property 'onSelect' are incompatible.
    Type '(event: SyntheticEvent<HTMLDivElement>, selection: string) => void' is not assignable to type '(event: SyntheticEvent<HTMLOptionElement>) => void'.
ERROR in /Users/priley/GitHub/web-ui/frontend/node_modules/@patternfly/react-core/dist/js/components/Select/Select.d.ts(13,30):
TS2304: Cannot find name 'ReactNode'.
ERROR in /Users/priley/GitHub/web-ui/frontend/node_modules/@patternfly/react-core/dist/js/components/Tabs/Tab.d.ts(4,14):
TS2304: Cannot find name 'node'.
ERROR in /Users/priley/GitHub/web-ui/frontend/node_modules/@patternfly/react-core/dist/js/components/Tabs/Tab.d.ts(9,19):
TS2304: Cannot find name 'elementType'.
ERROR in /Users/priley/GitHub/web-ui/frontend/node_modules/@patternfly/react-core/dist/js/components/Tabs/Tabs.d.ts(18,13):
TS2304: Cannot find name 'OneOf'.
```

--- a/packages/patternfly-4/react-core/src/components/EmptyState/EmptyState.d.ts
+++ b/packages/patternfly-4/react-core/src/components/EmptyState/EmptyState.d.ts
@@ -1,7 +1,7 @@
 import { FunctionComponent, HTMLProps, ReactNode } from 'react';
 import { OneOf, Omit } from '../../helpers/typeUtils';
 
-export const EmptyStateVariant = {
+export const EmptyStateVariant : {
   large: 'large',
   small: 'small',
   full: 'full'

--- a/packages/patternfly-4/react-core/src/components/Select/CheckboxSelectGroup.d.ts
+++ b/packages/patternfly-4/react-core/src/components/Select/CheckboxSelectGroup.d.ts
@@ -1,7 +1,7 @@
 import { HTMLProps, FormEvent } from 'react';
-import { Omit } from '../../typeUtils';
+import { Omit } from '../../helpers/typeUtils';
 
-export interface CheckboxSelectGroupProps extends Omit<HTMLProps<HTMLOptionElement>> {
+export interface CheckboxSelectGroupProps extends Omit<HTMLProps<HTMLOptionElement>, 'disabled'> {
   label?: string;
 }
 

--- a/packages/patternfly-4/react-core/src/components/Select/CheckboxSelectOption.d.ts
+++ b/packages/patternfly-4/react-core/src/components/Select/CheckboxSelectOption.d.ts
@@ -1,11 +1,10 @@
 import { HTMLProps, FormEvent } from 'react';
-import { Omit } from '../../typeUtils';
+import { Omit } from '../../helpers/typeUtils';
 
 export interface CheckboxSelectOptionProps extends Omit<HTMLProps<HTMLOptionElement>, 'disabled'> {
   value?: string;
   isDisabled?: boolean;
   isChecked?: boolean;
-  onClick?: Function;
 }
 
 declare const CheckboxSelectOption: React.FunctionComponent<CheckboxSelectOptionProps>;

--- a/packages/patternfly-4/react-core/src/components/Select/Select.d.ts
+++ b/packages/patternfly-4/react-core/src/components/Select/Select.d.ts
@@ -1,4 +1,4 @@
-import { HTMLProps, FormEvent } from 'react';
+import { HTMLProps, FormEvent, ReactNode } from 'react';
 
 export const SelectVariant: {
   single: 'single';
@@ -9,7 +9,6 @@ export interface SelectProps extends HTMLProps<HTMLOptionElement> {
   isExpanded?: boolean;
   isGrouped?: boolean;
   onToggle(value: boolean): void;
-  onSelect(event: React.SyntheticEvent<HTMLDivElement>, selection: string): void;
   placeholderText?: string | ReactNode;
   selections?: string | Array<string>;
   variant?: string;

--- a/packages/patternfly-4/react-core/src/components/Tabs/Tab.d.ts
+++ b/packages/patternfly-4/react-core/src/components/Tabs/Tab.d.ts
@@ -1,12 +1,13 @@
-import { FunctionComponent, HTMLProps } from 'react';
+import { FunctionComponent, HTMLProps, ReactNode } from 'react';
+import { Omit } from '../../helpers/typeUtils';
 
 export interface TabProps extends Omit<HTMLProps<HTMLDivElement>, 'id'> {
-  children?: node;
+  children?: ReactNode;
   className?: string;
   eventKey: number;
   id?: string;
-  tabContentId?: string|number;
-  tabContentRef?: elementType;
+  tabContentId?: string | number;
+  tabContentRef?: HTMLElement;
   title: string;
 }
 

--- a/packages/patternfly-4/react-core/src/components/Tabs/Tabs.d.ts
+++ b/packages/patternfly-4/react-core/src/components/Tabs/Tabs.d.ts
@@ -1,5 +1,5 @@
 import { FunctionComponent, HTMLProps, FormEvent } from 'react';
-import { Omit } from '../../helpers/typeUtils';
+import { OneOf, Omit } from '../../helpers/typeUtils';
 
 export const TabsVariant: {
   div: 'div';

